tests/rule_based_profiler/parameter_builder/test_parameter_computations.py

```diff
@@ -29,27 +32,136 @@ def _generate_distribution_samples(size: Optional[int] = 36) -> pd.DataFrame:
     return pd.DataFrame(data)
 
 
-def test_custom_bootstrap_efficacy():
+def test_legacy_bootstrap_point_estimate_efficacy():
     df: pd.DataFrame = _generate_distribution_samples(size=1000)
     false_positive_rate: np.float64 = np.float64(0.01)
     columns: pd.Index = df.columns
     column: str
-    lower_quantile: np.float64
-    upper_quantile: np.float64
+    lower_quantile_point_estimate: np.float64
+    upper_quantile_point_estimate: np.float64
     actual_false_positive_rates: Dict[str, Union[float, np.float64]] = {}
     for column in columns:
-        (lower_quantile, upper_quantile,) = compute_bootstrap_quantiles(
+        (
+            lower_quantile_point_estimate,
+            upper_quantile_point_estimate,
+        ) = _compute_bootstrap_quantiles_point_estimate_legacy(
             metric_values=df[column],
             false_positive_rate=false_positive_rate,
             n_resamples=DEFAULT_BOOTSTRAP_NUM_RESAMPLES,
         )
         actual_false_positive_rates[column] = (
             1.0
-            - np.sum(df[column].between(lower_quantile, upper_quantile)) / df.shape[0]
+            - np.sum(
+                df[column].between(
+                    lower_quantile_point_estimate, upper_quantile_point_estimate
+                )
+            )
+            / df.shape[0]
         )
         # Actual false-positives must be within 1% of desired (configured) false_positive_rate parameter value.
         assert (
             false_positive_rate - 0.01
             <= actual_false_positive_rates[column]
             <= false_positive_rate + 0.01
         )
+
+
+def test_bootstrap_bias_corrected_point_estimate_efficacy():
+    df: pd.DataFrame = _generate_distribution_samples(size=1000)
+    false_positive_rate: np.float64 = np.float64(0.01)
+    columns: pd.Index = df.columns
+    column: str
+    lower_quantile_bias_corrected_point_estimate: np.float64
+    upper_quantile_bias_corrected_point_estimate: np.float64
+    actual_false_positive_rates: Dict[str, Union[float, np.float64]] = {}
```

Comment:
@NathanFarmer Shall we use the `MetricValue` / `MetricValues` types we defined, or leave it as is?

Reply:
This is the target one-sided quantile for which to estimate the `MetricValue`. I feel like it should not be considered a `MetricValue`.